_posts/0000-01-02-EmyJoshy.md

```diff
@@ -0,0 +1,5 @@
+
+layout: slide
+title: "Welcome to our second slide!"
+
```

Comment:
Maintain your spirit of curiosity, keep questioning things and you will new find a new way to innovate.